packages/react-sdk/src/component-utils/image-loaders.ts

```diff
@@ -0,0 +1,48 @@
+import warnOnce from "warn-once";
+import { allSizes, type ImageLoader } from "./image";
+
+/**
+ * Default image loader in case of no loader provided
+ * https://developers.cloudflare.com/images/image-resizing/url-format/
+ **/
+export const cloudflareRelativeImageLoader: ImageLoader = ({
+  width,
+  src,
+  quality,
+}) => {
+  if (process.env.NODE_ENV !== "production") {
+    warnOnce(
+      !allSizes.includes(width),
```

Comment:
easier to scan, please trust me on this

```suggestion
      allSizes.includes(width) === false,
```